src/permutat.cc

```diff
@@ -1141,6 +1175,21 @@ static Obj FuncLARGEST_MOVED_POINT_PERM(Obj self, Obj perm)
     return INTOBJ_INT(LargestMovedPointPerm(perm));
 }
 
+/****************************************************************************
+**
+*F  FuncSMALLEST_MOVED_POINT_PERM( <self>, <perm> )
+*F  Smallest point moved by perm
```

Comment:
The `*F` is only for the function name, not for description.
```suggestion
**
**  Return the smallest point moved by <perm>, or Infinity.
```